docs/csharp/getting-started/introduction-to-the-csharp-language-and-the-net-framework.md

```diff
@@ -16,7 +16,7 @@ C# is an elegant and type-safe object-oriented language that enables developers
   
 ## C# Language
 
- C# syntax is highly expressive, yet it is also simple and easy to learn. The curly-brace syntax of C# will be instantly recognizable to anyone familiar with C, C++ or Java. Developers who know any of these languages are typically able to begin to work productively in C# within a very short time. C# syntax simplifies many of the complexities of C++ and provides powerful features such as nullable value types, enumerations, delegates, lambda expressions and direct memory access, which are not found in Java. C# supports generic methods and types, which provide increased type safety and performance, and iterators, which enable implementers of collection classes to define custom iteration behaviors that are simple to use by client code. Language-Integrated Query (LINQ) expressions make the strongly-typed query a first-class language construct.  
+ C# syntax is highly expressive, yet it is also simple and easy to learn. The curly-brace syntax of C# will be instantly recognizable to anyone familiar with C, C++ or Java. Developers who know any of these languages are typically able to begin to work productively in C# within a very short time. C# syntax simplifies many of the complexities of C++ and provides powerful features such as nullable value types, enumerations, delegates, lambda expressions, and direct memory access, which are not found in Java. C# supports generic methods and types, which provide increased type safety and performance, and iterators, which enable implementers of collection classes to define custom iteration behaviors that are simple to use by client code. Language-Integrated Query (LINQ) expressions make the strongly-typed query a first-class language construct.  
```

Comment:
Enumerations does exists in Java AFAIK.
For a reference, check [Java Enums - W3Schools](https://www.w3schools.com/java/java_enums.asp).

Lambda expressions were added to Java 8.
For reference, check [Lambda Expressions in Java 8 - GeeksforGeeks](https://www.geeksforgeeks.org/lambda-expressions-java-8/)

It's not good to refer to C# language features as non-existent in Java while they do.
This suggestions removes the features I found existed in Java. (I didn't look if other features being mentioned exists in Java.)

Another suggestions, is to just remove "which are not found in Java" and keep the Lambda expressions and Enumerations.

```suggestion
 C# syntax is highly expressive, yet it is also simple and easy to learn. The curly-brace syntax of C# will be instantly recognizable to anyone familiar with C, C++, or Java. Developers who know any of these languages are typically able to begin to work productively in C# within a very short time. C# syntax simplifies many of the complexities of C++ and provides powerful features such as nullable value types, delegates, and direct memory access, which are not found in Java. C# supports generic methods and types, which provide increased type safety and performance, and iterators, which enable implementers of collection classes to define custom iteration behaviors that are simple to use by client code. Language-Integrated Query (LINQ) expressions make the strongly-typed query a first-class language construct.  
```

Reply:
Thanks for noticing this. I just removed "which are not found in Java".